### PR TITLE
[breaking changes] fix(packages) create global entrypoint

### LINKED ISSUE #169: [deprecated] Agent.create move to global entrypoint
# TL;DR

Agent class initializer works on `constructor` for better

## Backward compatibility

want to no breaking changes.. from global

### PR DESCRIPTION
**Pull request by 9renpoto**
# TL;DR

Closes #169

### CODE REVIEW

File: packages/agent/src/global.ts [hunk 1]
```diff
@@ -1,24 +1,54 @@
...
-    }
-  })(window)
+const execute = (queue: any[], Agent: any): void => {
+  const { cmd, args, name } = parse(queue)
+  /**
+   * _ud('create', 'id', 'auto', 'myTracker')
+   * _ud('create', 'id', { name: 'myTracker' })
+   * _ud('create', 'id', 'auto')
+   */
+  if (cmd === 'create' && (name || args[1])) {
```
Comment (9renpoto) [inline packages/agent/src/global.ts]: https://developers.google.com/analytics/devguides/collection/analyticsjs/creating-trackers

Comment (9renpoto) [reply packages/agent/src/global.ts]: typeof args[1] !== 'string' && args[1].name === string

File: packages/agent/src/constants.ts [hunk 2]
```diff
@@ -15,13 +15,14 @@ export const NAMESPACE = `data-ud-namespace`
 export const SCROLL = ['pageXOffset', 'pageYOffset']
 export const TOUCH = ['ontouchstart', 'ontouchmove', 'ontouchend']
 export const VERSION = 1
-export const SETTINGS = {
+export const SETTINGS: Settings = {
+  allowLink: false,
+  linkerName: '__ud',
```
Comment (9renpoto) [inline packages/agent/src/constants.ts]: default query `__ud`

Comment (shoota) [reply packages/agent/src/constants.ts]: okey.

File: examples/built-in/index.pug [hunk 3]
```diff
@@ -0,0 +1,4 @@
+extends ../index.pug
+
+block footer_content
+  script(src='./built-in.bundle.js' async defer)
```
Comment (shoota) [inline examples/built-in/index.pug]: `../assets/build-in.bundle.js` ?

File: examples/built-in/index.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import Agent from '@userdive/agent'
+import Linker from '@userdive/linker'
+
+const agent = new Agent('af57h6gb', 'auto')
+agent.provide('linker', Linker)
+agent.require('linker')
+agent.run('linker', 'autoLink', ['developers.userdive.com'])
```
Comment (Surgo) [inline examples/built-in/index.ts]: ☺️

**Pull request merged**